### PR TITLE
build-chain tool removed from jenkins

### PR DESCRIPTION
**Pull request by Ginxo**
**Thank you for submitting this pull request**

**JIRA**: _(please edit the JIRA link if it exists)_ 

[link](https://examle.com)

**referenced Pull Requests**: _(please edit the URLs of referenced pullrequests if they exist)_

* https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/1808

<pre>
How to retest a PR or trigger a specific build:

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
</pre>

### CODE REVIEW

File: jenkins-agents/ansible/roles/kogito/tasks/main.yml [hunk 1]
```diff
@@ -28,9 +28,6 @@
 - name: Install Yarn
   shell: npm install -g yarn --registry=${NPM_REGISTRY_URL}
 
-- name: Install build-chain-action
-  shell: npm install -g @kie/build-chain-action@{{buildchain_version}} --registry=${NPM_REGISTRY_URL}
-
 - name: Install build-chain-action
```
Comment (mareknovotny) [inline jenkins-agents/ansible/roles/kogito/tasks/main.yml:31]: isn't this wrong name?

Comment (winklerm) [reply jenkins-agents/ansible/roles/kogito/tasks/main.yml:31]: +1 - not sure if it is related to build-chain-action (and removed as well) or just a wrong name for unrelated task

Comment (Ginxo) [reply jenkins-agents/ansible/roles/kogito/tasks/main.yml]: replaced by `Install npm-cli-login npm library` ! thanks

**Review by mareknovotny — APPROVED** [review]

**Comment by mareknovotny** [comment]
jenkins retest this please

**Comment by mbiarnes** [comment]
Jenkins retest this

**Comment by mbiarnes** [comment]
Jenkins retest this

**Pull request merged**